14/umbraco-cms/extending/property-editors/integrate-property-editors.md

```diff
@@ -0,0 +1,42 @@
+# Integrate Property Editors in your UI
+
+Property Editors can be used in any places of Backoffice and you can implement them too.
+
+## Property & Property Dataset Components
+
+To integrate one or more Property Editors, it can be done by using two Components.
+The Property Dataset component and a Property component.
```

Comment:
```suggestion
Integrating one or more Property Editors requires two Components: the Property Dataset component and a Property component.
```

Reply:
Well, it can be done in other ways. I understand its not so clear.. The point here is that this is the simplest way to implement it.

Reply:
.. and at this point, the only option we will be describing in our Docs. The other one is rather advanced.

Reply:
Aha! Could this be an option instead then:
"The simplest way to integrate one or more Property Editors is done using two Components: : the Property Dataset component and a Property component."

Reply:
Yes, rigtig godt :-)